extensions/wikia/PhalanxII/services/PhalanxService.class.php

```diff
@@ -278,15 +295,15 @@ private function sendToPhalanxDaemon( $action, $parameters ) {
 					$res = ( is_null( $response ) ) ? false : $response;
 					break;
 				case "status":
-					$res = ( stripos( $response, self::RES_STATUS  ) !== false ) ? true : false;
+					$res = ( stripos( $response, self::RES_STATUS ) !== false ) ? true : false;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "true" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)